### ISSUE TITLE
Quick Draw Shots spammable indefinitely if certain conditions are met?

### ISSUE THREAD

**Issue by VirtuosusFFXI** [EdenServer/community#2333]
### Checklist

- [x] I have searched for existing issues for issues like this one. The issue has not been posted. (Duplicate reports slow down development.)
- [ ] I have provided reproducable steps. (No "as the title says" posts please. Provide reproducable steps even if it seems like a waste of time.)
- [ ] I have provided my client version in the details. (type /ver into your game window)

### Details

One of our Corsairs in Dynamis - Xarcabard was able to spam Thunder Shot with no cooldown. We're not sure why it happened, but it only worked for him and no other CORs. He was still able to spam it even after dieing. 

![qd](https://user-images.githubusercontent.com/49970272/71609107-dee43880-2b53-11ea-942a-09b75f692d0a.png)

**Comment by Volkaru** [comment]
No other COR in that run was able to reproduce it.

**Comment by Tacoxtuesday** [comment]
Each shot added 1 min to the cooldown, after allowing the cooldown to fully reset, I was unable to fire multiple shots.
Version 30181205_0

**Comment by Caelic** [comment]
video of it happening:

https://www.youtube.com/watch?v=W3rTh19Aews

**Comment by KnowOne134** [comment]
Is this still a problem after patches on wed?

**Comment by Caelic** [comment]
@KnowOne134 We don't know how to reproduce it at all. All our other CORs during that dynamis weren't able to pull it off. I just know that it's got something to do with the "charges" system going whack even though its charge mechanic is effectively gone since it's just set to one now. Taco said it worked through both the JA menu and through macros.

**Comment by KnowOne134** [comment]
Gonna keep the issue open. Hopefully it has remedied itself with the patches. Post back here if it happens again. Anyone

**Comment by Caelic** [comment]
https://i.imgur.com/zbYWp3A.jpg

**Comment by RenaviOnEden** [comment]
Nivare is me btw, basically the target was slightly out of range and walking away from me, so i was spamming the macro so it'd go off asap.  it went off twice and once my Quick Draw CD was above 1m I was able to spam it as much as I wanted.  It added 1min CD each time to Quick Draw and didn't fix until I let the CD tick down to 0.

**Comment by Shoruto** [comment]
This is a copy and paste of Starved's report

**Title** - Infinite Quick Draw - COR #1764

**Details**

At the end of an level Synced party in Bhaflau Thickets, I used my 2hr Wild Card and noticed that the cooldown for Quick Draw was over 2:00 min (was 2:40). I thought this was an error with an addon and thought I may have had another shot left, but by spamming my macro, I found that I could endlessly use quickdraw and blasted a couple of birds with my new Ice Shot gattling gun to confirm the issue. It was fun while it lasted, but this should definitely be fixed before someone else finds and abuses, as it can probably be used to kill gods quite easily.

version 30181205_0
